website/docs/components/icon-tile/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,60 @@
+## Usage
+
+### When to use
+
+When displaying an object, or as part of a page title.
+
+### When not to use
+
+Inline in a string of text, such as a paragrph where an object is referred or linked to.
+
+## Color
+
+Use **Neutral** if the object or page is not a feature of a specific product, but is something universal. For example, for a "Dashboard" or "User" page.
+
+<Hds::IconTile @color="neutral" @icon="dashboard" />
+
+Use a **product-specific color** for objects or pages that are directly related to a product. For example:
+
+- For a page showing a "Consul cluster"
+- In a card or table row that represents a "Consul cluster"
+
+<Hds::IconTile @color="consul" @icon="server-cluster" />
+
+### Examples
+
+<div style="display: flex; gap: 2rem;">
+    <Hds::IconTile @color="neutral" @icon="user" />
+    <Hds::IconTile @color="boundary" @icon="crosshair" />
+    <Hds::IconTile @color="consul" @icon="server-cluster" />
+    <Hds::IconTile @color="nomad" @icon="briefcase" />
+    <Hds::IconTile @color="packer" @icon="layers" />
+    <Hds::IconTile @color="terraform" @icon="grid" />
+    <Hds::IconTile @color="vagrant" @icon="box" />
+    <Hds::IconTile @color="vault" @icon="key" />
+    <Hds::IconTile @color="waypoint" @icon="cloud-upload" />
+</div>
```

Comment:
Not sure if it's good to introduce inline styles like this. I assume we'd want to create something more reusable later on.

Reply:
Yeah, I agree and can remove for now. Wrapped some of the component examples, but we should use a consistent method so I'll strip out the inline styles for now.

Reply:
Came across similar needs in other components and wasn't sure how (or if) to style them